test/reducers/profile.spec.ts

```diff
@@ -1,19 +1,55 @@
 import { Action } from "../../src/actions";
 import { setUserMode } from "../../src/actions/profileActions";
 import { profileReducer } from "../../src/reducers/profileReducer";
-import { Profile, UserMode } from "../../src/state/profile";
+import { ApartmentType, GenderType, ProfileProps, UserRole } from "../../src/state/profile";
 
 describe("profile reducer", () => {
-  const inputState: Profile = {
-    basicDetails : "",
-    mode : UserMode.Tenant,
+  const inputState: ProfileProps = {
```

Comment:
And remove the code duplication here please 👍

Reply:
i dont think there are duplicates:
https://github.com/AccommoDateApp/accommodate-react/blob/444d6ed06a87246c33b0369fccef91772a217af0/test/reducers/profile.spec.ts